src/Kunstmaan/GeneratorBundle/Generator/PageGenerator.php

```diff
@@ -179,7 +180,7 @@ private function updateParentPages()
         foreach ($this->parentPages as $file) {
             $data = file_get_contents($file);
             $data = preg_replace(
-                '/(function\s*getPossibleChildTypes\s*\(\)\s*\{\s*return\s*array\s*\()/',
+                '/(function\s*getPossibleChildTypes\s*\(\)\s*\{\s*return\s*\[)/',
```

Comment:
Should we still allow `array` and `[` to keep BC with older classes?

Reply:
you're right, that would be better